packages/node/bin/autostaker-manual-test.sh

```diff
@@ -0,0 +1,52 @@
+# Do not merge this manual test to main
+
+NODE_PRIVATE_KEY="1111111111111111111111111111111111111111111111111111111111111111"
+OWNER_PRIVATE_KEY="2222222222222222222222222222222222222222222222222222222222222222"
```

Comment:
> We wouldn't like to require that operators provide their private key for this plugin. What other possibilities there are? The nodes can nowadays vote and flag behalf of the operator, so maybe they could also stake/unstake if that kind of permission could be added?

Nodes do node things; there's another role that can be added: CONTROLLER_ROLE. This is added by operator calling `operatorContract.grantRole("0x7b765e0e932d348852a6f810bfa1ab891e259123f02db8cdcde614c570223357", controllerAddress)` (where that hex mush can be read from `operatorContract.CONTROLLER_ROLE()`)

This is probably preferable as a key management mechanism, so that the autostaker would be run with a private key of this kind of "controller".